modules/sfp_haveibeenpwned.py

```diff
@@ -135,23 +179,54 @@ def handleEvent(self, event):
             self.results[eventData] = True
 
         data = self.query(eventData)
-        if data is None:
+        if data is not None:
+            for n in data:
+                try:
+                    if not self.opts['api_key']:
```

Comment:
Different fields depending on whether an API key is used or not? Isn't an API key always required for HIBP?